src/main/java/com/codegym/shopyy/model/Cart.java

```diff
@@ -0,0 +1,46 @@
+package com.codegym.shopyy.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "carts")
+@JsonIgnoreProperties({"user", "cartItems"})
+public class Cart {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+//    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
remove comments if not using or use // TODO: what you will do in the future

Reply:
to do: 
the commented-out code sections are intended to establish relationships with other entities in the future

Reply:
resoled